src/resources/triton_python_backend_utils.py

```diff
@@ -498,6 +500,27 @@ def add_output(self, output):
         self._model_config["output"].append(output)
 
 
+def log(message, level, nested_call=False):
+    logger = Logger()
+    logger.log(message, level, nested_call)
+
+
+def log_info(message):
+    log(message, Logger().INFO, True)
+
+
+def log_warn(message):
+    log(message, Logger().WARNING, True)
+
+
+def log_error(message):
+    log(message, Logger().ERROR, True)
+
+
+def log_verbose(message):
+    log(message, Logger().VERBOSE, True)
```

Comment:
It looks like we're instantiating a new logger for every single log and log().LEVEL here. Can there be some kind of global/singleton instead instantiated once?